src/index.ts

```diff
@@ -93,15 +93,15 @@ export enum BentoWindowEvents {
 export enum BentoDocumentEvents {
   /** Fired when a Button or CTA is clicked within Bento */
   buttonClicked = "bento-buttonClicked",
-  /** Fired when no available guide is found */
+  /** Fired when no available Guide is found */
   noGuideFound = "bento-noGuideFound",
-  /** Fired when a guide is loaded */
+  /** Fired when a Guide is loaded */
   onGuideLoad = "bento-onGuideLoad",
-  /** You can fire this event to tell Bento to close the sidebar component */
+  /** You can fire this event to tell Bento to close the Sidebar component */
   sidebarClose = "bento-sidebarClose",
-  /** You can fire this event to tell Bento to open the sidebar component, when available */
+  /** You can fire this event to tell Bento to open the Sidebar component, when available */
   sidebarOpen = "bento-sidebarOpen",
-  /** Fired when the Inline component is loaded */
+  /** Fired when the Inline (i.e. embedded card or checklist) component is loaded */
```

Comment:
This isn't true for embedded cards. For those, we fire an event containing the Guide Entity ID, therefore as far as I know it is impossible to add the types here. 

See #8345 for more details

Reply:
Cool, will leave it to you to modify. in my experience no one understands what "inline" means so clarifying we mean the onboarding checklist is likely helpful.